Xwt.XamMac/Xwt.Mac/ButtonBackend.cs

```diff
@@ -153,8 +153,12 @@ public void SetButtonType (ButtonType type)
 			}
 		}
 
+		public void SetFormattedText (FormattedText text)
+		{
+			Widget.Title = text.Text;
```

Comment:
Is it necessary to set the Title here? It will be overwritten by AttributedTitle in the next line.

Reply:
Actually, this never formats the selected item in the button properly anyway, so I can remove that line.